dockers/docker-ptf/Dockerfile.j2

```diff
@@ -96,8 +96,11 @@ RUN mkdir /var/run/sshd \
  && sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
  && sed -i '$aUseDNS no' /etc/ssh/sshd_config
 
-COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+COPY ["supervisord.conf", "/etc/supervisor/"]
+COPY ["conf.d/supervisord.conf", "conf.d/sshd.conf", "conf.d/ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+
+RUN mkdir -p /var/log/supervisor
 
 EXPOSE 22
 
-ENTRYPOINT ["/usr/bin/supervisord"]
+ENTRYPOINT ["/usr/local/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```

Comment:
Why do you need to specify the config file? As I mentioned above, the main config file, /etc/supervisor/supervisord.conf should include all conf files under /etc/supervisor/conf.d/. Is this not working?